app/views/dispatch/forecast.erb

```diff
@@ -0,0 +1,65 @@
+<div class="container">
+
+	<div class='row'>
+		<div class='col-xs-12'>
+			<div class='panel panel-default'>
+				<div class='panel-heading'>
+					Active Missions
+				</div>
+				<% if @active_missions.length == 0 %>
+					<div class='panel-body'>
+						<h5>No trucks out</h5>
+					</div>
+				<% else %>
+					<% @active_missions.each do |mission| %>
+						<div class='panel-body'>
+							<p class="text-success">
+								<%= "#{mission.dateTime} :: #{mission.name} #{mission.unit_serviced}" %>
```

Comment:
this should also be extracted out into a helper since we are formatting the information about the mission